addons/huntir/subConfig/config.cpp

```diff
@@ -12,6 +11,8 @@ class CfgPatches {
         author = ECSTRING(common,ACETeam);
         url = ECSTRING(main,URL);
         VERSION_CONFIG;
+
+        addonRootClass = QUOTE(COMPONENT);
```

Comment:
in game it's `addonRootClass = "huntir";` for all of these

Reply:
That's what it should be.

Reply:
```suggestion
        addonRootClass = QUOTE(ADDON);
```

Reply:
Oh! Right... whoever did the compats subconfigs did that too, eg. https://github.com/acemod/ACE3/blob/56e905ab7a0ba951f21b15fa8007386866be9aef/addons/compat_gm/compat_gm_refuel/config.cpp#L5